src/Symfony/Component/HttpKernel/Tests/Fragment/InlineFragmentRendererTest.php

```diff
@@ -39,6 +39,9 @@ public function testRenderWithControllerReference()
         $this->assertEquals('foo', $strategy->render(new ControllerReference('main_controller', [], []), Request::create('/'))->getContent());
     }
 
+    /**
+     * @group time-sensitive
```

Comment:
Can't we add this on the whole class instead?

Reply:
done